caput/config.py

```diff
@@ -264,55 +290,54 @@ def _finalise_config(self):
         To be overridden in subclasses if we need to perform some processing
         post configuration.
         """
+        ...
 
 
-def utc_time(default=None):
+def utc_time(default: TimeLike | None = None) -> Property[TimeLike]:
     """Property for representing UTC as UNIX time.
```

Comment:
```suggestion
    """Property type for representing UTC as UNIX time.
```
For consistency with other `Property`-returning functions.